### PR TITLE
Global styles: output :root selector for CSS custom properties and remove unused vars

### PR DESCRIPTION
**Pull request by ramonjd**
Synching:

- https://github.com/WordPress/gutenberg/pull/42084
- https://github.com/WordPress/gutenberg/pull/59938

Trac ticket: https://core.trac.wordpress.org/ticket/61135

### CODE REVIEW

File: src/wp-includes/class-wp-duotone.php [hunk 1]
```diff
@@ -814,7 +814,7 @@ private static function get_svg_definitions( $sources ) {
 	 * @return string The CSS for global styles.
 	 */
 	private static function get_global_styles_presets( $sources ) {
-		$css = 'body{';
```
Comment (tellthemachines) [inline src/wp-includes/class-wp-duotone.php:817]: Might be worth adding `@since` comment up there with the change?

**Review by tellthemachines — APPROVED** [review]
Thanks for the PR! Tiny comment below, otherwise LGTM!

File: src/wp-includes/class-wp-duotone.php (hunk 1 above)
Comment (ramonjd) [reply src/wp-includes/class-wp-duotone.php:817]: Oh good spotting. Thanks! I'll fix.

**Comment by tellthemachines** [comment]
Committed in [r58123](https://core.trac.wordpress.org/changeset/58123).

**Pull request closed without merging**